pkg/deploy/devconfig.go

```diff
@@ -48,6 +48,7 @@ func deployKeyvaultAccessPolicy(_env env.Core) map[string]interface{} {
 		"objectId": os.Getenv("AZURE_SERVICE_PRINCIPAL_ID"),
 		"permissions": map[string]interface{}{
 			"secrets": []interface{}{
+				"Get",
```

Comment:
no - i think the policy is already set [here](https://github.com/Azure/ARO-RP/blob/master/pkg/deploy/generator/resources_rp.go#L1211-L1212)

Good catch - double checking the testing.

Reply:
Ok spent some time thinking about this now and I remember.  

This change here is intended.  We need to get individual secrets during deploy to check how long ago they were created to ensure we don't create a new secret if the secret is less than 3 days (72 hours) old.  These secrets that we're checking are in both the service keyvault and the portal keyvault shown [here](https://github.com/Azure/ARO-RP/pull/1645/files#diff-9c46df97568b8e9865152bf3f58fa6f8feb1b4d7f734c14ba7d5fc7d2d311f54R398-R400).  

This is only for the dev-config because the deployer SP needs permission to read that.  There's a matching set of updates to the Fairfax and public cloud INT/Prod instances to the RP config in this [RP Config PR](https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/RP-Config/commit/3087e76f8a96b296a64cfa43e0421582ccfd4b4c?refName=refs/heads/encryption-rotation-keyvault-perms&path=/deploy/prod-config.yaml)

I did miss the additional portal access policies within the PR, but have now updated them as a result.